### DISCUSSION TITLE
Plans to support other Qualcomm DSP environments?

### DISCUSSION THREAD

**Discussion by icebeing** [halide/Halide#8479] [General]
HI, 

I recently implemented some Halide algorithms for Qualcomm's Hexagon vNN target.  Are there plans to support other types of DSP cores, like the Tensilica Hifi [1-5] offering, via the Qualcomm S7 Audio Platform?

Thanks, Charles.

**Comment by alexreinking** [comment]
That would be a question for @aankit-ca or @adasgupt1
